crates/zkvm/lib/src/ecdsa/projective.rs

```diff
@@ -0,0 +1,430 @@
+//! Implementation of the SP1 accelerated projective point.
+//!
+//! This type is mainly used in the `ecdsa-core` algorithms.
+//!
+//! Note: SP1 uses affine arithmetic for all operations.
```

Comment:
```suggestion
//! Note: When performing curve operations, accelerated crates for SP1 use affine arithmetic instead of projective arithmetic for performance.
```